src/components/MaterialUI/Inputs/RadioProps.js

```diff
@@ -20,6 +21,7 @@ class RadioProps extends ComponentProps {
 		this.componentProps.set(this.constructor.propNames.row, true);
 		this.componentProps.set(this.constructor.propNames.name, null);
 		this.componentProps.set(this.constructor.propNames.radios, null);
+		this.componentProps.set(this.constructor.propNames.disabled, false);
```

Comment:
don't define default values inside props. it should be defined directly inside component.